editor.planx.uk/src/@planx/components/SetValue/utils.ts

```diff
@@ -0,0 +1,89 @@
+import { Store } from "pages/FlowEditor/lib/store";
+import { SetValue } from "./model";
+
+type PreviousValues = string | string[] | undefined;
+
+type HandleSetValue = (params: {
+  nodeData: SetValue;
+  previousValues: PreviousValues;
+  passport: Store.passport;
+}) => Store.passport;
+
+/**
+ * Handle modifying passport values when passing through a SetValue component
+ * Called by computePassport()
+ */
+export const handleSetValue: HandleSetValue = ({
```

Comment:
Following [previous conversations](https://github.com/theopensystemslab/planx-new/pull/2914#discussion_r1534124245), I've refactored this logic to be co-located with the component rather than being purely within the store.

The motivations here were two-fold - 
 - This is SetValue logic and doesn't relate to any other components - it just can't get called by this component directly
 - We actually want to use some of this code within the Editor modal to show the Editor examples of how this component works (upcoming PR).